tests/xtro-sharpie/ios.pending

```diff
@@ -46,6 +46,16 @@
 ## fixed for XAMCORE_4_0
 !incorrect-protocol-member! QLPreviewItem::previewItemTitle is OPTIONAL and should NOT be abstract
 
+# UserNotifications
+
+## Shorter enum names
```

Comment:
not sure about those... we don't know the future of that framework and they could become confusing in the future (even if right now I like them better).

in any case a type rename should be added in the comments of the binding files (to ease audits) and this should also move to the `Helpers.cs` file